sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -586,6 +587,122 @@ case class DescribeTableCommand(
   }
 }
 
+/**
+ * Command that looks like
+ * {{{
+ *   DESCRIBE [EXTENDED|FORMATTED] table_name column_name;
+ * }}}
+ */
+case class DescribeColumnCommand(
+    table: TableIdentifier,
+    column: String,
+    isExtended: Boolean,
+    isFormatted: Boolean)
+  extends RunnableCommand {
+
+  override val output: Seq[Attribute] =
+    // Column names are based on Hive.
```

Comment:
I got these names by running hive. I can't find any document about the names, but I'll add a link of the corresponding JIRA of Hive.